tip_tracker/lib/core/auth/login/LobbyPage.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:tip_tracker/core/auth/login/Log_in_form.dart';
+import 'package:tip_tracker/core/auth/login/Sign_up_form.dart';
+
+class LobbyPage extends StatefulWidget {
+  const LobbyPage({Key? key}) : super(key: key);
+
+  @override
+  State<LobbyPage> createState() => _LobbyPageState();
+}
+
+class _LobbyPageState extends State<LobbyPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xff183A37),
+      body: SafeArea(
+        child: Center(
+          child: Column(children: [
+            //LOGO
+            const SizedBox(height: 149),
+            const Text(
+              'TIPMATE',
+              style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 64,
+                  color: Color(0xffEFD6AC)),
+            ), //Text
+
+            //Sign in Button
+            const SizedBox(
+              height: 128,
+            ),
+            GestureDetector(
+              //Handle Gesture
+              onTap: () {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(builder: (context) => const Log_in_form()),
+                );
+              },
+
+              child: Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 25.0),
+                child: Container(
+                  padding: const EdgeInsets.fromLTRB(16, 12, 12, 16),
+                  decoration: BoxDecoration(
+                      border: Border.all(color: const Color(0xff0BFF4F)),
```

Comment:
For these hex codes I would assign them to some variable describing what this color is just for more clarity.

Reply:
you can create a new file under lib/config/themes name something like colors.dart, create a class Colors and start creating a bunch of static variables for each color you want to use.